README.md

```diff
@@ -111,6 +111,13 @@ celery -A openlcsd worker --loglevel=INFO
 celery -A openlcsd worker --loglevel=DEBUG
 ```
 
+### Running OpenLCS beat service
+OpenLCS beat service can be started using the following (from the top directory of
+the repo):
+```shell
+cd openlcs && celery -A openlcs beat --loglevel=INFO
```

Comment:
`cd openlcs`, maybe we change to use `in the directory as manage.py`

Reply:
Do u mean` celery -A openlcs.openlcs beat --loglevel=INFO` or i misunderstand.